src/ng/directive/ngRepeat.js

```diff
@@ -302,6 +304,8 @@ var ngRepeatDirective = ['$parse', '$animate', function($parse, $animate) {
     // jshint bitwise: false
     scope.$odd = !(scope.$even = (index&1) === 0);
     // jshint bitwise: true
+    scope.$prev = prev;
+    scope.$next = next;
```

Comment:
So, I can understand why you'd want this feature, but we do see a real performance impact from adding properties to each scope, it's not ideal. I think we need to be very careful about adding new ones